contracts/utils/StrikeSelection.sol

```diff
@@ -152,6 +153,49 @@ contract StrikeSelection is Ownable {
         }
     }
 
+    /**
+     * @notice Gets the strike price by multiplying the current spot price with a multiplier
+     * @param expiryTimestamp is the unix timestamp of expiration
+     * @param isPut is whether option is put or call
+     * @param strikeMultiplier is the multiplier to compute the strike price
+     * @return newStrikePrice is the strike price of the option (ex: for BTC might be 45000 * 10 ** 8)
+     * @return newDelta is the delta of the option given its parameters
+     */
+
+    function getSimpleStrikePrice(
```

Comment:
Instead of writing a new function, we should just create a new contract that uses the new methodology. Call it `PercentStrikeSelection.sol` and the current one `DeltaStrikeSelection.sol`. This helps us switch between the multiple strike selection methodologies